haystack/telemetry_2.py

```diff
@@ -185,26 +190,26 @@ def send_pipeline_run_event(  # type: ignore
             event_properties["pipeline.run_parameters.params"] = bool(params)
             event_properties["pipeline.run_parameters.debug"] = bool(debug)
 
-            telemetry.send_event(event_name=event_name, event_properties=event_properties)
+            telemetry.send_event(event_name="Pipeline run", event_properties=event_properties)
```

Comment:
Any reason why this isn't just `Pipeline` (like `Training` or `Evaluation`)?

Reply:
Good question. I would like to be able to tell training, evaluation and inference clearly, but I could do that with an event property too. One benefit of using the event name is that we can't possibly count training and evaluation with inference, they are always going to be clearly separate. On the other hand I don't expect training and eval to be very relevant in terms of event count. So if you prefer I can use a property for this.

Reply:
It's fine, no need to change! 👍